lib/active_merchant/billing/gateways/pin.rb

```diff
@@ -206,7 +215,11 @@ def unparsable_response(raw_response)
       end
 
       def token(response)
-        response['token']
+        if response['token'].start_with?('cus')
+          "#{response.dig('card', 'token')}+#{response['token']}"
```

Comment:
There's no real standardization, but unless you think `+` is particularly suited for this gateway, let's use one of the more common separators, either `;` or `|`.

Reply:
Also, since prior tokens only consisted of the customer token, that should be the first element so that splitting old tokens will still parse them correctly.

Reply:
+1 for using a common separator.

Reply:
👍